app/views/users/login_form.html.erb

```diff
@@ -4,16 +4,17 @@
     <h2>Log In</h2> 
   </div>
 
-<div id="form">
-  <div class="input">
-    <%= form_with url: login_path, method: :post do |form| %>
-      <%= form.label :email, "Email:" %>
-      <%= form.email_field :email, required: true, type: "email" %>
-      <br>
-      <%= form.label :password, "Password:" %>
-      <%= form.password_field :password, required: true, minlength: 6, type: "password" %>
-      <br>
-      <%= form.submit "Log In", class: "submit" %>
-    <% end %>
-  </div>
+
+<div class="input">
+  <%= form_with url: login_path, method: :post do |form| %>
+    <%= form.label :email, "Email:" %>
+    <%= form.email_field :email, required: true, type: "email" %>
+    <br>
+    <%= form.label :password, "Password:" %>
+    <%= form.password_field :password, required: true, minlength: 6, type: "password" %>
+    <br>
+    <%= form.submit "Log In", class: "submit" %>
+  <% end %>
+  <br>
+  <%= button_to "Log in with Google", '/auth/google_oauth2', class: "submit", data: { turbo: false } %>
```

Comment:
Awesome button. Maybe we will refactor it later with one of those images we added